index.js

```diff
@@ -0,0 +1,37 @@
+window.addEventListener('DOMContentLoaded', () => {
+
+    const popupImages = document.querySelectorAll('.featured-img__style');
+    const popup = document.querySelector('.popup');
+
+    popupImages.forEach((img, index) => {
+        img.addEventListener('click', (img) => {
+            document.querySelector('.popup-img').src = img.target.src;
+        });
+        img.addEventListener('click', openPopup);
+    });
+
+    function openPopup() {
+        popup.classList.add('show-popup');
+        popup.classList.remove('hiden-popup');
+        document.body.style.overflow = 'hidden';

```

Comment:
Why do you need to use it here?

Reply:
Because without "document.body.style.overflow = ### 'hidden'" the page will scroll when the modal is open

Reply:
Thanks for question!
Because without "document.body.style.overflow ='hidden'" the page will scroll when the modal is open

Reply:
You can add this in `css`. Most of the time you should prefer to use `css` instead of `javasript` if it possible :)

Reply:
> You can add this in `css`. Most of the time you should prefer to use `css` instead of `javasript` if it's possible :)

ahhh, got it, thanks a lot)